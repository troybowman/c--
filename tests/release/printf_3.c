../decl/printf_3.c