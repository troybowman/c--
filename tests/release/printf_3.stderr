error, line 4: ellipsis "..." notation is is only valid when declaring builtin function: extern void printf(char format[], ...);
c-- returned nonzero exit code